test/unit/org/apache/cassandra/io/sstable/CQLSSTableWriterConcurrencyTest.java

```diff
@@ -70,6 +72,7 @@ public void testConcurrentSchemaModification() throws InterruptedException, IOEx
         File[] dataDirs = new File[nThreads];
         String baseDataDir = tempFolder.newFolder().getAbsolutePath();
 
+        AtomicReference<String> errors = new AtomicReference<>("");
```

Comment:
what i don't get, this is passing on trunk but failing constantly on accord... what did accord change to make this unstable?